__main__.py

```diff
@@ -54,7 +54,12 @@ def main(
     """
     if config_path is None:
         pkd_dir = Path(__file__).resolve().parent
-        config_path = str(pkd_dir / "run_config.yml")
+        if Path(pkd_dir / "pipeline.yml").is_file():
+            config_path = str(pkd_dir / "pipeline.yml")
+        elif Path(pkd_dir / "run_config.yml").is_file():
+            config_path = str(pkd_dir / "run_config.yml")
+        else:
+            config_path = str(pkd_dir / "pipeline.yml")
```

Comment:
Will 
```suggestion
        if not (pkd_dir / "pipeline.yml").is_file():
            config_path = str(pkd_dir / "run_config.yml")
        else:
            config_path = str(pkd_dir / "pipeline.yml")
```
achieve the same result?

`pkd_dir` is already a `Path` so I don't think we have to cast it again

Reply:
Will remove the redundant casting of `pkd_dir`.

The logic the original code implements is:
1. If `pipeline.yml` present, pick it as default.
2. Else if user has old `run_config.yml`, pick it for backward compatibility.
3. If both are not found, default to new `pipeline.yml`.

The suggested amended code will change 3. to default to the old `run_config.yml`.